scss/block/alert_banner.scss

```diff
@@ -0,0 +1,16 @@
+//import
+@import '../abstracts/mixins';
+@import '../abstracts/variables';
+
+.block--type-alert-banner {
+  margin-bottom: 30px;
+  .block-title {
+    display: none;
```

Comment:
lets not hide the Title via code, as we can hide it via the Block Configuration.